app/lua/lua.c

```diff
@@ -602,12 +455,11 @@ static bool readline(lua_Load *load){
         {
           /* Get a empty line, then go to get a new line */
           c_puts(load->prmt);
-          continue;
         } else {
           load->done = 1;
           need_dojob = true;
-          break;
```

Comment:
I can't figure out what is going on here....

Reply:
The original logic is -- how should i put it -- tangled.  To be honest I can't remember why I removed the break and instead fell through the the following continue.    

What the original code did was to break on receiving an EOL whereas this continue to receive another char.  I'll try both variants out and see which works the best.

Whatever, we've got whole blocks of code `#if 0`ed out since being hacked out for the first project commit.  IMO, if they don't do anything and will never be reintroduced then they should be removed so we can see the wood for the trees.